front/app/containers/Admin/messaging/messages.ts

```diff
@@ -264,6 +264,40 @@ export default defineMessages({
     id: 'app.containers.Admin.emails.groups',
     defaultMessage: 'Groups',
   },
+  viewExample: {
+    id: 'app.containers.Admin.emails.viewExample',
+    defaultMessage: 'View',
+  },
+  noExampleYet: {
+    id: 'app.containers.Admin.emails.noExampleYet',
+    defaultMessage:
+      'We can’t show you an example email as no email of this type has been sent yet. As soon as one is sent you’ll be able to view it here.',
+  },
```

Comment:
```suggestion
  noExampleYet: {
    id: 'app.containers.Admin.emails.noExampleYet1',
    defaultMessage:
      "We can't show you an example email as no email of this type has been sent yet. As soon as one is sent you'll be able to view it here.",
  },
```

It's the weird Figma quote again.